emails/build-emails

```diff
@@ -5,12 +5,20 @@ const fs = require('fs')
 const path = require('path')
 const prettier = require('prettier')
 const handlebars = require('handlebars')
-const { MOCK_DATA, PRINT_DATA_FOR } = require('./env')
+const postmark = require('postmark')
+const { 
+  MOCK_DATA, 
+  PRINT_DATA_FOR,
+  SYNC_TEMPLATES, 
+  POSTMARK_SERVER_API_TOKEN
+} = require('./env')
 
 const DIST_DIR = path.resolve('dist')
 
-function renderTemplate(createTemplate, name) {
-  let { template, templateText, mockData } = createTemplate()
+function renderTemplate(filename) {
+  const name = path.basename(filename, '.js')
```

Comment:
It looks like we may want to sanitize the current template names in Postmark first, so we don't get a bunch of duplicates from name mismatches :).

Reply:
I'm following a slightly different alias/naming so I will just manually delete old ones when we merge. You can already see new templates in postmark, I've ran the template sync manually.